flashcards/backend/decks/models.py

```diff
@@ -20,3 +20,24 @@ def __str__(self):
     class Meta:
         verbose_name = "category"
         verbose_name_plural = "categories"
+
+
+class DifficultyLevel(models.Model):
+    """Model for representing how hard flashcard/deck is to learn
+
+    :param name: level name
+    :type name: str
+    :param value: value
+    :type value: int
+
+    """
+
+    name = models.CharField(max_length=20)
+    value = models.IntegerField()
```

Comment:
diff lvl can be negative numbers? i suggest changing it to PositiveIntegerField

Reply:
Normaly in games you would have, 0 for 'normal', -1 for easy, -2 for 'very easy' , -10 for 'even your dog can do that' and so on. But I can have it changed if you like.

Reply:
Even if we'd wanted negative values for difficulty, we do not need to store it in a 32-bit field, 16-bit is more than enough I think. I side with @MWasile here 😅

Reply:
PositiveSmallIntegerField it is then, thanks for input.